Automattic/woocommerce-payments
# Update the Fraud & Risk tools welcome tour copy

Fixes #6155

#### Changes proposed in this Pull Request

Update the Fraud & Risk tools welcome tour copy to match the latest design changes.









#### Testing instructions





* The unit tests should be passing
* Run `npm run wp option add wcpay_fraud_protection_settings_active 1` to enable the feature flag
* Run `npm run wp option delete wcpay_fraud_protection_welcome_tour_dismissed` to reset the welcome tour flag
* The component layout should match the Figma design ([WIM5w2Y10SCNBFdMLSVftj-fi-3696_268417](WIM5w2Y10SCNBFdMLSVftj-fi-3696_269486))

-------------------


**Post merge**
